src/style.css

```diff
@@ -0,0 +1,120 @@
+@font-face {
+  font-family: "Avenir";
+  src: url(./fonts/Avenir-Book.ttf);
+  font-style: normal;
+  font-weight: 300;
+}
+
+@font-face {
+  font-family: "Avenir";
+  src: url(./fonts/Avenir-Heavy.ttf);
+  font-style: normal;
+  font-weight: 800;
+}
+
+html {
+  font-weight: 300;
+}
+
+.search__large {
+  margin-top: 20px;
+  width: 100%;
+  height: 70px;
+  box-sizing: border-box;
+  border: 1px solid #e1e7ed;
+  border-radius: 4px;
+  background: #fff;
```

Comment:
color '#fff' is duplicated, so please move all duplicated styles to the variables